rls/src/main/java/io/grpc/rls/CachingRlsLbClient.java

```diff
@@ -968,11 +968,11 @@ public PickResult pickSubchannel(PickSubchannelArgs args) {
         // Happy path
         logger.log(ChannelLogLevel.DEBUG, "Returning PickResult");
         PickResult pickResult = picker.pickSubchannel(args);
-        // TODO: include the "grpc.target" label once target is available here.
         if (pickResult.hasResult()) {
           helper.getMetricRecorder().addLongCounter(TARGET_PICKS_COUNTER, 1,
-              Lists.newArrayList("", lookupService, childPolicyWrapper.getTarget(),
-                  determineMetricsPickResult(pickResult)), Lists.newArrayList());
+              Lists.newArrayList(helper.getChannelTarget(), lookupService,
+                  childPolicyWrapper.getTarget(), determineMetricsPickResult(pickResult)),
+              Lists.newArrayList());
```

Comment:
We can do as a follow-up: Use Collections.emptyList(), since it doesn't need to create a new list every time? Lists.newArrayList() returns a mutable list, so it isn't reused.

FWIW, Arrays.asList() is probably overall better than `newArrayList()` for most cases in our code, unless you need a list you can append to.

Reply:
I just made the change here.